api/http/playground.go

```diff
@@ -0,0 +1,18 @@
+//go:build playground
```

Comment:
todo: Should add a copyright header (I am also wondering why the linter didn't complain about the copyright header)

Reply:
Maybe due to the build tag here? I'm not exactly sure either.